src/EditorFeatures/Core/Options/ColorSchemeOptions.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Composition;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Options.Providers;
+
+namespace Microsoft.CodeAnalysis.Editor.Options
+{
+    internal static class ColorSchemeOptions
+    {
+        public const string SettingKey = "TextEditor.Roslyn.ColorScheme";
+
+        public const string Enhanced = nameof(Enhanced);
```

Comment:
I feel like I set us up for failure with this one. I assume because of our work the C++ team has adopted the scheme names "Visual Studio 2017" and "Enhanced". Which I now feel compelled to match. I think the way out would be for unified cross-language "scheme" support in the platform. At which time someone can reset all the names to something more sensible.

Reply:
If it is incremental improvements to the enhanced theme then I don't think there is a need for a new scheme. If there is a new scenario that we wanted to use highlighting to imrpove (C++ team have an "Enhanced Globals vs Locals" theme i believe) then we could name it something appropriate.